sdk/schemaregistry/Azure.Data.SchemaRegistry/src/SchemaRegistryClient.cs

```diff
@@ -116,11 +116,11 @@ await RegisterSchemaInternalAsync(groupName, schemaName, schemaDefinition, forma
                 ResponseWithHeaders<SchemaRegisterHeaders> response;
                 if (async)
                 {
-                    response = await RestClient.RegisterAsync(groupName, schemaName, new BinaryData(schemaDefinition).ToStream(), cancellationToken).ConfigureAwait(false);
+                    response = await RestClient.RegisterAsync(groupName, schemaName, format.ContentType, new BinaryData(schemaDefinition).ToStream(), cancellationToken).ConfigureAwait(false);
```

Comment:
out of curiosity: format has a public `ContentType` instance variable..? I didn't remember see that in the .Net api view